src/platforms/android/androidcontroller.cpp

```diff
@@ -181,6 +181,27 @@ void AndroidController::activate(const HopConnection& hop, const Device* device,
     excludedApps.append(QJsonValue(appID));
   }
 
+  // Find a Server as Fallback in the Same Location in case
```

Comment:
This is a pretty neat idea, and I like it better than the silent server switching we currently have. Although such a change is probably out of scope for the size of this PR, I would be interested to know if we could generalize the "fallback server" into something that's supported on all platforms.